openmdao/drivers/case_driver.py

```diff
@@ -0,0 +1,34 @@
+"""
+OpenMDAO driver that runs a user-specified list of cases.
+"""
+
+from openmdao.drivers.predeterminedruns_driver import PredeterminedRunsDriver
+
+
+class CaseDriver(PredeterminedRunsDriver):
+    """OpenMDAO driver that runs a sequence of cases.
+
+    Args
+    ----
+    cases : sequence of cases
+        A sequence of cases, where each case is a sequence of (name, value) tuples.
+
+    num_par_doe : int, optional
+        The number of cases to run concurrently.  Defaults to 1.
```

Comment:
I think you're right, but not sure if I want to change it now since there are people out there using other PredeterminedRunsDrivers and they all use num_par_doe.